ledger-bridge.js

```diff
@@ -140,11 +140,11 @@ export default class LedgerBridge {
         }
     }
 
-    async signTransaction (replyAction, hdPath, tx, to) {
+    async signTransaction (replyAction, hdPath, tx, to, chainId) {
         try {
             await this.makeApp()
             if (to) {
-                const isKnownERC20Token = byContractAddress(to)
+                const isKnownERC20Token = byContractAddress ? byContractAddress(to) : byContractAddressAndChainId(to, Number(chainId))
```

Comment:
Doing this `byContractAddress` check because the `gh-pages` changes happen *immediately* upon merge; this ensures that the `gh-pages` code works between the branch being merged and updated release of the extension.

Reply:
Hmmm... not sure if I understand.

This PR updates the version of `@ledgerhq/hw-app-eth` to which has `byContractAddressAndChainId`. Why does the release of the extension have any affect on this?

Reply:
^ Yeah, this seems like a mistake. Maybe you meant to check for the existence of `chainId`? That would be omitted on the older extension version I would expect.

Reply:
Good catch, I confused myself there.